fields.go

```diff
@@ -69,13 +69,13 @@ type Commit struct {
 	Revert      *Revert // If it is not a revert commit, `nil` is assigned
 	Refs        []*Ref
 	Notes       []*Note
-	Mentions    []string // Name of the user included in the commit header or body
+	Mentions    []string   // Name of the user included in the commit header or body
 	JiraIssue   *JiraIssue // If no issue id found in header, `nil` is assigned
-	Header      string   // (e.g. `feat(core)[RNWY-310]: Add new feature`)
-	Type        string   // (e.g. `feat`)
-	Scope       string   // (e.g. `core`)
-	Subject     string   // (e.g. `Add new feature`)
-	JiraIssueId string   // (e.g. `RNWY-310`)
+	Header      string     // (e.g. `feat(core)[RNWY-310]: Add new feature`)
+	Type        string     // (e.g. `feat`)
+	Scope       string     // (e.g. `core`)
+	Subject     string     // (e.g. `Add new feature`)
+	JiraIssueID string     // (e.g. `RNWY-310`)
```

Comment:
This is a `BREAKING CHANGE` as the original role out of the Jira integration expected `JiraIssueId` as the type in the config.